api/base/pagination.py

```diff
@@ -41,6 +41,10 @@ def page_number_query(self, url, page_number):
 
         return paginated_url
 
+    def get_self_real_link(self, url):
+        page_number = self.page.number
```

Comment:
`self` just links to the current "page"

![screen shot 2016-09-27 at 1 43 48 pm](https://cloud.githubusercontent.com/assets/7913604/18885160/7ab1d740-84b8-11e6-9012-d26968870101.png)